spec/lib/mdot/client_spec.rb

```diff
@@ -144,17 +169,9 @@
     end
 
     context 'with an malformed order' do
-      it 'returns a 400 error' do
-        VCR.use_cassette('mdot/submit_order_400') do
-          expect(StatsD).to receive(:increment).once.with(
-            'api.mdot.submit_order.fail', tags: [
-              'error:Common::Client::Errors::ClientError', 'status:400'
-            ]
-          )
-          expect(StatsD).to receive(:increment).once.with('api.mdot.submit_order.total')
-          set_mdot_token_for(user)
-          expect { subject.submit_order({}) }.to raise_error(MDOT::ServiceException)
-        end
+      it 'returns a 422 error' do
```

Comment:
Could you add a spec for `MDOT_invalid` and `MDOT_service_unavailable`?

Reply:
Done and done